README.md

```diff
@@ -78,13 +78,27 @@ cd kubermatic
 ```
 
 There are a couple of scripts in the `hacks` directory to aid in running the components locally
-for testing purposes.
+for testing purposes. You can create a cluster via the UI at `https://dev.kubermatic.io`.
```

Comment:
I feel uncomfortable to bring back all those references to our dev system into this public README. This sentence would only hold true for Kubermatic employees and so I don't think it belongs here. In general we are moving away from relying on dev for everything.

Reply:
I see where you are coming from. Removed the ref.